simple_email_confirmation/models.py

```diff
@@ -292,3 +299,12 @@ def auto_add(sender, **kwargs):
     #       get_user_model() here - results in import loop.
 
     post_save.connect(auto_add)
+
+
+def get_email_address_model():
```

Comment:
Could we move this to `__init__.py` like in `django.contrib.auth`?

Reply:
👍  Yep.